packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWRadioPanel/CWRadioPanel.tsx

```diff
@@ -0,0 +1,66 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { CWText } from 'views/components/component_kit/cw_text';
+import CWIconButton from 'views/components/component_kit/new_designs/CWIconButton';
+import CWPopover, {
+  CWPopoverProps,
+  usePopover,
+} from 'views/components/component_kit/new_designs/CWPopover';
+import { CWRadioButton } from 'views/components/component_kit/new_designs/cw_radio_button';
+import { ComponentType } from 'views/components/component_kit/types';
+
+import './CWRadioPanel.scss';
+
+interface CWRadioPanelProps<T> {
+  value: T;
+  popover?: Pick<CWPopoverProps, 'title' | 'body'>;
+  label: string;
+  description: string;
+  isSelected: boolean;
+  onSelect: (value: T) => void;
+}
+
+const CWRadioPanel = <T,>({
+  value,
+  label,
+  description,
+  popover,
+  isSelected,
+  onSelect,
+}: CWRadioPanelProps<T>) => {
+  const popoverProps = usePopover();
+
+  return (
+    <div
+      className={clsx(ComponentType.RadioPanel, { selected: isSelected })}
+      onClick={() => onSelect(value)}
+    >
```

Comment:
maybe we could add this as a button for better accessibility (lighthouse also complains about clickable elements should be buttons)

Reply:
makes sense, will change that in upcoming PR